src/ui/mod.rs

```diff
@@ -1,16 +1,20 @@
+mod login;
+
 use anyhow::Result;
 use console::Term;
 
 use crate::command::HELP_STR;
 use crate::service;
 use crate::{Command, Resources, ResourcesProvider};
 
-#[allow(unused)] // FIXME
 pub struct Ui<R: ResourcesProvider> {
     res: Resources<R>,
     term: Term,
 }
 
+static USERNAME_STR: &str = "Username: ";
+static PASSWORD_STR: &str = "Password: ";
+
```

Comment:
Regarding these, I think we can just define the strings inline (in `prompt_username` and `prompt_password`). String translations are not in the radar I assume.

I put the help string in a `static` so that it could be written in the same place as the command line arguments are parsed.

Reply:
I thought that defining most UI strings as `static`/`const` in one same place would make the code more organized but either way is fine.